game/platform.cc

```diff
@@ -1,17 +1,17 @@
 #include "platform.hh"
 #include "fs.hh"
 
-Platform::platforms Platform::currentOS() {
-if (BOOST_OS_WINDOWS != 0) { return windows; }
-else if (BOOST_OS_LINUX != 0) { return linux; }
-else if (BOOST_OS_MACOS != 0) { return macos; }
-else if (BOOST_OS_BSD != 0) { return bsd; }
-else if (BOOST_OS_SOLARIS != 0) { return solaris; }
-else if (BOOST_OS_UNIX != 0) { return unix; }
+Platform::HostOS Platform::currentOS() {
+if (BOOST_OS_WINDOWS != 0) { return PERFORMOUS_WIN; }
```

Comment:
Hmmm where are these `BOOST_OS_X` coming from? i guess since we link with boost? Is there a way in `std` to correctly determine OS?

Aside from that this is pretty much a rename so that's alright.

Reply:
There isn't, or wasn't last I checked.
These are part of the Boost.predef